sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpProperty.cs

```diff
@@ -31,6 +37,24 @@ internal static class AmqpProperty
         ///
         public static AmqpSymbol TrackLastEnqueuedEventProperties { get; } = AmqpConstants.Vendor + ":enable-receiver-runtime-metric";
 
+        /// <summary>
+        ///   The capability for opting-into idempotent publishing.
+        /// </summary>
+        ///
+        public static AmqpSymbol EnableIdempotentPublishing { get; } = AmqpConstants.Vendor + ":idempotent-producer";
+
+        /// <summary>
+        ///   The identifier of the producer group to associate with a producer.
+        /// </summary>
+        ///
+        public static AmqpSymbol ProducerGroupId { get; } = AmqpConstants.Vendor + ":producer-sequence-number";
+
+        /// <summary>
+        ///   The sequence number assigned by a producer to an event when it was published.
+        /// </summary>
+        ///
+        public static AmqpSymbol PublishedSequenceNumber { get; } = AmqpConstants.Vendor + ":producer-id";
```

Comment:
Are these values transposed?

Reply:
They are;  I've got a fix for this in the next set of changes, so I'm going to punt the fix to the next PR.